API/API.mli

```diff
@@ -4635,8 +4588,6 @@ sig
   val add_suffix : Names.Id.t -> string -> Names.Id.t
   val increment_subscript : Names.Id.t -> Names.Id.t
   val make_ident : string -> int option -> Names.Id.t
-  val out_name : Names.Name.t -> Names.Id.t
-  val pr_lab : Names.Label.t -> Pp.std_ppcmds
```

Comment:
No. The comment was just about `pr_lab`.

The comment about `out_name` can be found above.